pages/usage.md

```diff
@@ -0,0 +1,106 @@
+# Usage
+
+`boringproxy` consists of two kinds of long running processes that act as server and client.
+
+Run it in the shell, with `bg`/`fg`, in a tmux session, or configure a system daemon.
+
+It is managed via an administrative web interface.
+
+---
+
+## Command line interface
+
+The command line interface is used to create server and client instances.
+
+## Server
+
+The server requires you to pass the domain it will serve. It is invoked with:
+
+> `./boringproxy server -admin-domain <admin-domain>`
+
+> `-admin-domain` - the main domain configured in DNS
+
+Example:
+
+```
+boringproxy server -admin-domain bpdemo.brng.pro
+```
+
+Upon first run a configuration file `boringproxy_db.json` with admin credentials will be created and a login link is displayed that looks like:
+
+> `https://<FQDN>/login?access_token=<TOKEN>`
+
+Example:
+
+```
+https://bpdemo.brng.pro/login?access_token=yJaicLl6zj48zZItXvtQGb4CH5m5fId5
+```
+
+## Client
+
+The client needs to know which server to connect to, a valid combination of user and access token plus a unique name for the current client. You can start it with:
+
+```bash
+./boringproxy client \
+    -server <server> -user <user> -token <token> -client-name <client-name>
+```
+
+> `-server` - the domain of the remote server  
+> `-user` - username, min. six characters, if not admin  
+> `-token` - access token to authenticate against the service  
+> `-client-name` - identifier for the client, used in the interface to distinguish the available tunnel origins  
+
+Example:
+
+```
+boringproxy client \
+    -server bpdemo.brng.pro \
+    -token fKFIjefKDFLEFijKDFJKELJF \
+    -client-name demo-client \
+    -user demo-user
+```
+
+---
+
+## Web interface
+
+The web interface is used to manage users, access tokens and tunnels.
+
+Access it at `https://<FQDN>/` where you will be presented with a prompt for a token, or with the direct login link from above.
+
+> ![](./../screenshot.png)
```

Comment:
I try to avoid adding binary files to git. Especially ones like this where we'll want to update it over time as the UI improves. We can handle the screenshot the same way we do the video files. I'll just make sure a copy of it is available to the production copy of the site. The path location is fine.

Unfortunately I'm not sure if there's an easy way to remove it from git at this point. Even if you add a commit that removes it, it will still be in the repo history. It's not a huge deal since it's only 55k, so if you don't know an easy way to do it don't worry about it.

Reply:
There were complicated ways to remove files from the history, but https://github.com/newren/git-filter-repo makes everything very easy these days.